### ISSUE TITLE
Information about FID metric to evaluate the model

### ISSUE THREAD

**Issue by rinodipaola15** [lucidrains/denoising-diffusion-pytorch#68]
I trained the network using the training images of Cifar10's 'horse' class (5000 samples).
I want to compute FID metric to evaluate the model, computing the "distance" between my generated images and dataset images.
To do this, do I have to pass, in the FID formula, the train dataset, the test dataset or both?

!python pytorch_fid path1 path2
path1 = /generated_images_folder
path2 = ?

**Comment by greens007** [comment]
> I trained the network using the training images of Cifar10's 'horse' class (5000 samples). I want to compute FID metric to evaluate the model, computing the "distance" between my generated images and dataset images. To do this, do I have to pass, in the FID formula, the train dataset, the test dataset or both?
> 
> !python pytorch_fid path1 path2 path1 = /generated_images_folder path2 = ?
> 
> Ps. Congratulations for your excellent work!

According to the paper, you must calculate FID between your generated images and the train dataset

**Comment by pimakshay** [comment]
path1 = /generated_images_folder (fake images)
path2 = /testing_dataset (real images)
